site/content/docs/main/config/gateway-api.md

```diff
@@ -198,7 +198,8 @@ containers:
   - --contour-key-file=/certs/tls.key
   - --config-path=/config/contour.yaml
   - --disable-feature=tlsroutes
-  - --disable-feature=grpcroutes
```

Comment:
hm, WDYT should we no longer let this resource be disabled? maybe in a future release? doesn't really hurt to let users still disable it tbh

Reply:
Hmm yeah that's a fair point, no strong opinion either way TBH but given the pattern is to only allow disabling alpha APIs, could make sense to remove it from that list now. Alternately, I'd be fine saying that _all_ non-essential resources can be disabled, so users can pare things down as much as they want.

Reply:
> all non-essential resources can be disabled

yeah seems reasonable to leave it for now, we could just open an issue to see if we want to make it required when using gw api since it is v1, but probably want to wait a release to make it required since it requires some operator action to make sure the crd is installed/rbac etc.